less/mixins.less

```diff
@@ -1 +1,9 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.btn-mixin(@color, @tColor, @fSize){
+    text-align: center;
+    color: @tColor;
+    background-color: @color;
+    font-size: @fSize;
+    padding: 15px;
+    border-radius: 10px;
+}
```

Comment:
great mixin
You could also try adding a hover effect to improve UI of the buttons.